config/core/configmaps/data-residency.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-data-residency
```

Comment:
I suggest rename this configmap to something like `pubsub-config`. I can imagine there will be other pubsub settings could fit here, e.g. CMEK.

Reply:
The data residency requirements might be used other than pubsub in the future, though it seems that we are only using pubsub as the storage now. So I would say we should keep it.

Reply:
Good point. We probably should revisit this idea if we find that we don't need to configure other services for data residency.